### PR TITLE
Tooltips on action conversation buttons

### PR DESCRIPTION
**Pull request by jsilvestre**
* Added structure to manage tooltips with aria-tips lib
* Added tooltips (see #138)
* Also very minor style change that was annoying me: the outline on the button to expand hidden messages in a conversation.

### CODE REVIEW

**Comment by frankrousseau** [comment]
@jsilvestre did you see with  @m4dz before doing it? I mentioned a lib he built to do it.

**Comment by frankrousseau** [comment]
Whatever this PR lacks of comments

**Comment by jsilvestre** [comment]
aria-tips is @m4dz's lib. I'm adding comments.

**Comment by jsilvestre** [comment]
I added more comments and built the code.

File: client/app/locales/en.coffee [hunk 1]
```diff
@@ -359,3 +359,14 @@ module.exports =
       'calendar unknown format' : """
             This message contains an invite to an event in a currently unknown format.
             """
+
+      # Tooltips
+      "tooltip reply": "Answer"
+      "tooltip reply all": "Answer to all"
+      "tooltip forward": "Forward"
+      "tooltip remove message": "Remove"
+      "tooltip open attachments": "Open attachments list"
```
Comment (frankrousseau) [inline client/app/locales/en.coffee]: There is no `s` at `attachment`. This word has an epithet value.

Comment (jsilvestre) [inline client/app/locales/en.coffee]: Weird, so you say "attachment list", but you also say "a list of attachments"?

Comment (frankrousseau) [inline client/app/locales/en.coffee]: Yes, I think it works that way.

**Pull request merged**